virttest/libvirt_installer.py

```diff
@@ -0,0 +1,125 @@
+"""
+Installer code that implement KVM specific bits.
+
+See BaseInstaller class in base_installer.py for interface details.
+"""
+
+import os
+import platform
+from avocado.utils import process
+import base_installer
+
+
+__all__ = ['GitRepoInstaller', 'LocalSourceDirInstaller',
+           'LocalSourceTarInstaller', 'RemoteSourceTarInstaller']
+
+
+class LIBVIRTBaseInstaller(base_installer.BaseInstaller):
+
+    '''
+    Base class for libvirt installations
+    '''
+
+    def _set_install_prefix(self):
+        """
+        Prefix for installation of application built from source
+
+        When installing virtualization software from *source*, this is where
+        the resulting binaries will be installed. Usually this is the value
+        passed to the configure script, ie: ./configure --prefix=<value>
+        """
+        prefix = self.test_builddir
+        self.install_prefix = os.path.abspath(prefix)
+
+    def _install_phase_package(self):
+        """
+        Create libvirt package
+        """
+        if os.path.isdir("/root/rpmbuild/"):
```

Comment:
It is not good to hardcode path.

Reply:
let me add a param with default value to it...